src/help/guides/adding-repos.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Adding Repositories
+lang: en-US
+---
+
+# External Source Repositories
+## What is an External Repository?
+**Paperback** out of the box only comes with **MangaDex** as a source for manga. The app however has support for adding manga from external repositories which are not on the official listings. Beta Sources, 18+ Sources, and sources developed independently are seperated due to **Paperback** having to adhere to Apple policies in order to be distributed through TestFlight and the Apple App Store. 
+
+::: warning Warning
+Currently external sources are only available on the Beta version of the application. If you are not a Patreon, this guide will not be very useful to you. This feature will be available in public builds in the future.
+:::
+
+
+## Adding an External Repository
+1. Open up Paperback on your iOS device
+1. Navigate to the Settings tab of the application
+1. Navigate to the Sources tab inside of Settings
+1. At the bottom of the page, you are able to add a repository by specifying a name, and the URL which the repository sits at. Ensure that your repository title is at least 4 characters long.
+    > Reference the [Known Repositories](/help/guides/adding-repos/#known-repositories) section for a list of known repositories, their links, and what they contain
+1. Tap 'Add'
```

Comment:
Maybe using `"` would be more consistent with getting-started guide: `"Install AltStore"`
```suggestion
1. Tap "Add"
```